src/components/TipInput.vue

```diff
@@ -130,25 +130,33 @@ export default {
       return (this.value > 0 && (this.isRetip || this.isMessageValid)) || this.isMessageValid;
     },
     isTipped() {
-      return !this.loading
-        || (this.tip.sender === this.account)
-        || this.tip.retips.filter((retip) => retip.sender === this.account).length > 0;
+      if (!this.stats || !this.derivedTipStats) {
+        return false;
+      }
+      return this.derivedTipStats.senders.find((sender) => sender === this.account);
     },
     iconTip() {
       return this.isTipped ? iconTipped : iconTip;
     },
+    derivedTipStats() {
+      if (!this.stats || !this.stats.by_url) {
+        return false;
+      }
+      if (this.isRetip) {
+        return this.stats.by_url.find((tipStats) => tipStats.url === this.tip.url);
+      }
+      return this.stats.by_url.find((tipStats) => tipStats.url === `${window.location.origin}/#/tip/${this.tip.id}`);
```

Comment:
should probably use a static url for superhero, as otherwise the content breaks if the url changes

Reply:
https://beta.superhero.com/#/tip/${this.tip.id} is this what you mean?
I guess it will be harder to test it.

Reply:
probably the final final url or an abstract representation @mradkov

Reply:
I thought of having a prefix or custom schema: e.g. `superhero://tip/123`

But this wont be an correct URL then and oracles would require tweaking.

Reply:
we are in control what we open? so superhero:// can be adjusted to local environment before opening

Reply:
oracles need adjustment anyways in order to properly support this feature

Reply:
I suggest we use `beta.superhero.com` for now. When sh is moved to the final URL we can adjust it. (still tips on beta will be able to be claimed).